### PR TITLE
Tensorflow implementation of number theory - s4494112

### PR DESCRIPTION
**Pull request by Emily-yan1402**
Implemented Euclidean algorithm and extended Euclidean algorithm in number theory. See README.md for more description.

Letian Yan
44941125

### CODE REVIEW

File: number_theory/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Tensorflow Implementation of Number Theory
+
+#### This is the readme document for COMP3710 project report.
+The implementation contains modules of Euclidean algorithm (number\_theory\_gcd.py) and extended Euclidean algorithm (number\_theory\_xgcd.py) which requires tensorflow 1.15.0 to run. 
+
+
+### Algorithm description
```
Comment (shakes76) [inline number_theory/README.md]: Needs depends list

File: number_theory/number_theory_xgcd.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""number_theory_xgcd.ipynb
```
Comment (shakes76) [inline number_theory/number_theory_xgcd.py]: Good

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Requested Feedback. Remove unnecessary file**
Request Description minimal. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Comment by Emily-yan1402** [comment]
DS_store file removed. Thanks for the feedback.

**Comment by shakes76** [comment]
Thanks

**Pull request merged**